docs/Advanced Scenarios for Qcow2 to OVA.md

```diff
@@ -0,0 +1,18 @@
+# Scenarios
+## Scenario 1: Create a small 20GB image for testing
+
+```shell
+$ pvsadm image qcow2ova  --image-name rhel-83-12182020  --image-url ./rhel-8.3-ppc64le-kvm.qcow2 --image-dist rhel --rhn-user jsmith --rhn-password re@llyASt0ngRHNPass0rd --image-size 20
+```
+
+## Scenario 2: Create an image with user selected root password for debug purpose
+
+```shell
+$ pvsadm image qcow2ova  --image-name rhel-83-12182020  --image-url ./rhel-8.3-ppc64le-kvm.qcow2 --image-dist rhel --rhn-user jsmith --rhn-password re@llyASt0ngRHNPass0rd --os-password someEasyPassword
+```
+
+## Scenario 3: Use the user defined directory for the temp directory(place used for image conversion)
+
+```shell
+$ pvsadm image qcow2ova  --image-name rhel-83-12182020  --image-url ./rhel-8.3-ppc64le-kvm.qcow2 --image-dist rhel --rhn-user jsmith --rhn-password re@llyASt0ngRHNPass0rd --temp-dir /home/jsmith
+```
```

Comment:
Maybe we can add another advanced scenario 4 to use -k flag inline for supplying the cloud api key incase users dont want to export the key.

Reply:
api key is not required for this command, its needed only for the upload and import.